base/initdefs.jl

```diff
@@ -371,6 +371,21 @@ function set_active_project(projfile::Union{AbstractString,Nothing})
     end
 end
 
+"""
+    active_manifest()
+    active_manifest(project_file::AbstractString)
+
+Return the path of the active manifest file, or the manifest file that would be used for a given `project_file`.
+See [`Project environments`](@ref project-environments) for details on the difference between a project and a manifest, and the naming
+options and their priority in package loading.
+
+See also [`Base.active_project`](@ref), [`Base.set_active_project`](@ref).
+"""
+function active_manifest(project_file::Union{AbstractString,Nothing}=nothing, search_load_path::Bool=true)
+    project_file === @something project_file active_project(search_load_path)
```

Comment:
```suggestion
    # If `project_file` was specified, use that, otherwise get the active project:
    project_file === !isnothing(project_file) ? project_file : active_project(search_load_path)
```